### PR TITLE
add sync srcipt

### PR DESCRIPTION
**Pull request by LongfeiH**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] [local build the book](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [ ] review the artifacts from local build
- [ ] [local build the slides](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [x] Unit test if appliable

# Checklist:

- [x] My code follows the [style guidelines](https://github.com/ocademy-ai/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have applied tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,4 +23,5 @@ open-machine-learning-jupyter-book/assignments/prerequisites/*
 */tmp/**
 */**/node_modules/*
 */node_modules/*
-awesome/database/node_modules
\ No newline at end of file
+awesome/database/node_modules
```
Comment (zhangqi444) [inline .gitignore:26]: Let's call it as .config.development.json. You may also need a prod one.

**Review by zhangqi444 — COMMENTED** [review]
move the new added files into `awesome/sync`

File: .gitignore (hunk 1 above)
Comment (LongfeiH) [reply .gitignore]: done

File: awesome/package.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
```
Comment (zhangqi444) [inline awesome/package.json:2]: such file should be in the sync folder right?

**Review by zhangqi444 — APPROVED** [review]

**Pull request merged**